src/viam/app/data_client.py

```diff
@@ -114,8 +114,9 @@ async def main():
         if __name__ == '__main__':
             asyncio.run(main())
 
-    """
 
+    For more information, see `Data Client <https://docs.viam.com/appendix/apis/data-client/>`_.
```

Comment:
(minor) Above the comment is "For more information, see {FOO} Client _API_", but here we just say "see Data Client". For consistency, should this be "Data Client API"?

Reply:
changed